lib/analyzer.rb

```diff
@@ -1,11 +1,13 @@
 EXERCISES = %w[
   two_fer
   acronym
+  leap
 ].freeze
 
 FILENAMES = {
   "acronym" => "acronym.rb",
-  "two-fer" => "two_fer.rb"
+  "two-fer" => "two_fer.rb",
+  "leap" => "leap.rb"
```

Comment:
Consider adding a `,` so that the addition is self contained for the next maintenance event, providing for a cleaner diff.  (Optional, of course).

Reply:
Done

Reply:
Rubocop didn't like this change either.

Reply:
Rubocop locally on your side, or our Rubocop configuration?

Reply:
I am using the repo's rubocop configuration locally. It failed in CI when I made that change.

Reply:
Thanks.

Reply:
The tooling `bin/rubocop-quick` is helpful for that, for sure.  It also needs some work, I created #106 to address one point talked about here, and another for the general maintenance that is due for the script.